### DISCUSSION TITLE
Explore: improvements to Loki log row context

### DISCUSSION THREAD

**Discussion by dprokop** [grafana/grafana#42314] [Observability]
- [ ] Enable loading log row context independently in either direction
- [ ] Add simple search input in context view to enable quick filtering of the context
- [ ] Display labels used for the context query
- [ ] Manipulate context query labels, to filter out fewer log streams (https://github.com/grafana/grafana/pull/17097#issuecomment-493405413)
- [ ] Log row context should render timestamps in context view

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
